src/showcase/SubscribeForm/index.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withTheme } from 'styled-components';
+import Input from '../Input';
+import Button from '../Button';
+
+class SubscribeForm extends React.Component {
+
+  constructor(props){
+    super(props);
+    this.state = {
+      placeholder: 'Enter your email',
+      buttonText: 'Subscribe',
+      isButtonDisabled: false,
+    }
+    this.handleInputChange = this.handleInputChange.bind(this);
+    this.handleButtonClick = this.handleButtonClick.bind(this);
+  }
+  
+  buttonRef = React.createRef();
+
+  validateEmail(email) {
+    const regexp = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+    return regexp.test(email);
+  }
+
+  handleInputChange(e){
+    this.setState({
+      value: e.target.value
+    });
+  }
+
+  handleButtonClick(e){
+    console.log(this.state.value);
+    if (!this.validateEmail(this.state.value)){
+      this.setState({
+        value: '',
+        placeholder: 'Invalid email. Try again'
+      });
+    } else {
+        var xhr = new XMLHttpRequest();
+        xhr.open("POST", 'https://cors-anywhere.herokuapp.com/https://mailchimp.mybit.io/api/member/', true);
```

Comment:
Haven't tested the code yet but I'm guessing this URL is wrong :)

Reply:
It isn't :). For me, it works.
Apparently, I needed to disable CORS, and that is impossible via { mode: 'no-cors' } in the browser, so I used a proxy-url.

Reply:
Oh I see, but we can't merge this as is.

Reply:
I don't see another way. CORS cannot be set from the browser, so another alternative would be to modify your [server](https://github.com/MyBitFoundation/mailchimp-proxy/blob/master/index.js).

Reply:
Thanks for the update! No need to disable CORS, we can probably have a staging URL that doesn’t has it, but the component should only be used in *.mybit.io domains. My suggestion is to have the URL passed down as a parameter or set out in a dynamic way depending in a env variable like `NODE_ENV`

Reply:
Thanks! @csmartinsfct can you take a look at this and see if we can test it locally?

Reply:
Hey @csmartinsfct did you manage to test this? Also, @jjperezaguinaga, unfortunately, I didn't receive the bounty for https://github.com/MyBitFoundation/MyBit-Network.tech/issues/151 (the pull request had "Fixes #151" in the description). Is there any way to get the lost MYB's?

Reply:
👋 @uivlis Apologies for that! Just commented on that issue, pretty much open a PR with an empty commit, so you get access to the bounty ☺️